packages/db/src/collection/index.ts

```diff
@@ -450,6 +450,62 @@ export class CollectionImpl<
     return this.config.getKey(item)
   }
 
+  /**
+   * Get a stable view key for a given item key.
+   * If viewKey configuration is enabled, returns the stable viewKey.
+   * Otherwise, returns the key as a string (backward compatible behavior).
+   *
+   * @param key - The item key to get the view key for
+   * @returns The stable view key as a string
+   *
+   * @example
+   * // Use in React components for stable keys during ID transitions
+   * {todos.map((todo) => (
+   *   <li key={collection.getViewKey(todo.id)}>
+   *     {todo.text}
+   *   </li>
+   * ))}
+   */
+  public getViewKey(key: TKey): string {
+    const viewKey = this._state.viewKeyMap.get(key)
+    return viewKey ?? String(key)
```

Comment:
Keys are of type `TKey = string | number`. If we have no `viewKey` we turn the key into a string and use it as the `viewKey`. Is there any reason to have keys as `string | number` but `viewKey`s always as `string`? Would it make sense to also allow `viewKey`s to be of type `string | number` ?